versions.tf

```
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.12.20, < 0.16"
+  required_version = ">= 0.12.20, ~> 1.0.0"
```

this will limit it to `1.0.x` and not add support.

```suggestion
  required_version = ">= 0.12.20, < 1.1.0"
```

---

I have tested this and it works with Terraform 1.0.2

See https://www.terraform.io/docs/language/expressions/version-constraints.html#gt--1

---

Ah I understand the point you are making, this negates the `>=0.12.20`